monitoring/cloud-client/src/main/java/com/example/monitoring/Snippets.java

```diff
@@ -392,6 +394,17 @@ void listMonitoredResources() throws IOException {
     // [END monitoring_list_resources]
   }
 
+  // [START monitoring_get_resource]
+  void getMonitoredResource(String resourceId) throws IOException {
+    String projectId = System.getProperty("projectId");
+    MetricServiceClient client = MetricServiceClient.create();
+    MonitoredResourceDescriptorName name = MonitoredResourceDescriptorName.of(projectId,
```

Comment:
Is there a better way to wrap this line?

Reply:
+1

Reply:
Done, thanks for catching.